decoder/candidates_test.go

```diff
@@ -421,7 +421,7 @@ func TestDecoder_CandidatesAtPos_rightHandSide(t *testing.T) {
 		},
 	}
 	testConfig := []byte(`myblock "foo" {
-  num_attr = 
+  num_attr =
```

Comment:
Was this intentional?

Reply:
Nope. Do not have `"files.trimTrailingWhitespace": true,` and edit a file, then decide not to do something with that file 😂